File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.py
T = int(input())
for x in range(T):
    H, W, N = map(int, input().split())
    X = N % H
    Y = N // H + 1
    if X == 0:
        X = H
        Y = N // H
    print(f"{X}{Y:02}")

        

File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.cs
using System;

class Program
{
    static void Main()
    {
        int t = int.Parse(Console.ReadLine());  // 테스트 개수
        
        for (int a = 0; a < t; a++)
        {
            string[] input = Console.ReadLine().Split();
            
            int H = int.Parse(input[0]);
            int W = int.Parse(input[1]);
            int N = int.Parse(input[2]);
            
            int floor = N % H;
            int room = N / H + 1;
            
            if (floor == 0)
            {
                floor = H;
                room = N / H;
            }
            
            Console.WriteLine($"{floor}{room:00}");
        }
    }
}
